src/2024h2/user-wide-cache.md

```diff
@@ -0,0 +1,121 @@
+# User-wide build cache
+
+| Metadata |                                                              |
+| -------- | ------------------------------------------------------------ |
+| Owner(s) | ![Help wanted][]    |
+| Teams    | [cargo]           |
+| Status   | Orphaned            |
```

Comment:
Not sure on the exact process for these fields in accepting the goal and having it in the RFC if we're working under the assumption that there is funding available and contractors that can pick it up.

Reply:
I'm not sure either. I think you should do one of two things

* if you have a concrete owner in mind, and they are interested in doing the contracting, put them here
* if you don't yet know for sure or don't want to name anyone, keep it how you have it

Either way, you should put yourself or some other specific people down as a mentor (in the section below).

Reply:
I think I'll skip naming for now